### PR TITLE
Dex 207 assets from wbia

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Tidied up some junk left over from previous PR
- Mostly just deleting the Annotations get all passthrough to WBIA but tidied up some trace output too

---
**Pull Request Checklist**
- [x] Ensure that the PR is properly formatted
  - Example: All lint checks are passing
- [x] Ensure that the PR is properly rebased
  - Example: The PR is rebased on `develop` (commit: `<insert develop commit hash>`)
- [x] Ensure that the PR uses a consolidated database migration
  - Example: One database migration is proposed (revision `<insert develop revision>` -> `<insert new revision>`)
- [x] Ensure that the PR is properly tested
  - Example: All automated tests are passing
- [x] Ensure that the PR is properly covered
  - Example: The percentage of the code covered by tests has not decreased
- [x] Ensure that the PR is properly sanitized
  - Example: No sensitive data or large content was added to this PR
- [ ] Ensure that the PR is properly reviewed

### CODE REVIEW

File: app/extensions/acm/__init__.py [hunk 1]
```diff
@@ -30,13 +33,116 @@ class ACMManager(RestManager):
             'list': '//annot/json/',
             'data': '//annot/name/uuid/json/?annot_uuid_list=[{"__UUID__": "%s"}]',
         },
+        'assets': {
+            'list': '//image/json/',
+            # TODO this is wrong but it's not clear what the correct option is
```
Comment (hwindsor) [inline app/extensions/acm/__init__.py:38]: Help!!!

File: app/modules/assets/models.py [hunk 2]
```diff
@@ -16,6 +18,74 @@
...
+    def ensure_acm_obj(cls, guid):
+        """
+        Doesn't ensure that an object exists, creates a new one to perform the sync
+        """
+        assetSync = AssetSync(guid=guid)
+
+        return assetSync
+
+    def acm_sync_complete(self):
+        # TODO update existing asset or create the new asset from the populated data in AssetSync
```
Comment (hwindsor) [inline app/modules/assets/models.py:70]: Thoughts on this? 
Asset has loads of columns that are nullable=False so we can't create an empty one for population.
Also where does the Submission data come from? AFAIK ACM knows nothing about the submission and neither does EDM.

File: app/extensions/edm/__init__.py [hunk 3]
```diff
@@ -123,13 +115,8 @@ def edm_sync_all(cls, verbose=True, refresh=False):
...
                 updated_items.append(model_obj)
             except sqlalchemy.exc.IntegrityError:
-                log.error(
-                    'Error updating %s %r'
-                    % (
-                        cls.EDM_NAME,
-                        model_obj,
-                    )
-                )
+                log.error(f'Error updating {cls.EDM_NAME} {model_obj}')
```
Comment (naknomum) [inline app/extensions/edm/__init__.py:118]: been loving the new `f'xxxxx'` format thing btw, using it all the time.

**Review by naknomum — APPROVED** [review]
lgtm

**Pull request merged**